packages/windows/install-mroonga.ps1

```diff
@@ -4,18 +4,42 @@ Param(
 )
 
 function Wait-UntilRunning($cmdName) {
+  $Waiting = $TRUE
   do
   {
-    $Running = Get-Process $cmdName -ErrorAction SilentlyContinue
-    Start-Sleep -m 500
-  } while (!$Running)
+    switch ($cmdName) {
+      "mysqld" {
+        if (Test-Path ..\mysqld.txt) {
+          $Version = Get-Content ..\mysqld.txt | Select-String -Pattern "^Version:"
+          if ($Version) {
+            $Waiting = $FALSE
+          }
+        }
+      }

```

Comment:
This is depends on how to run `mysqld`. So we should not use general name such as `Wait-UntilRunnning`.

We should define `Run-MySQL` instead of using `Start-Process` and `Wait-UntilRunning`.

Reply:
Thanks, I've fixed to use extracted named function.